packages/frontend/src/pages/ParkingSpaces/components/CloseListing.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react'
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  Typography,
+  Box,
+  IconButton,
+} from '@mui/material'
+import CloseIcon from '@mui/icons-material/Close'
+import { toast } from 'react-toastify'
+import { LoadingButton } from '@mui/lab'
+
+import { useCloseParkingSpaceListing } from '../hooks/useCloseParkingSpaceListing'
+
+export const CloseListing = (props: { listingId: number }) => {
+  const closeListing = useCloseParkingSpaceListing()
+  const [open, setOpen] = useState(false)
+
+  const onClose = () => {
+    setOpen(false)
+    closeListing.reset()
+  }
+
+  const onCloseListing = () =>
+    closeListing.mutate(
+      { listingId: props.listingId },
+      {
+        onSuccess: () => {
+          setOpen(false)
+          toast('Bilplatsannons markerad som ompublicerad', {
+            type: 'success',
+            hideProgressBar: true,
+          })
+        },
+      }
+    )
+
+  return (
+    <>
+      <Button variant="dark" onClick={() => setOpen(true)}>
+        Markera som publicerad
+      </Button>
+      <Dialog
+        onClose={onClose}
+        open={open}
+        maxWidth="xs"
+        TransitionProps={{ exit: false }}
+      >
+        <Box paddingTop="0.5rem">
+          <Box display="flex">
+            <DialogTitle variant="h1" fontSize={24} textAlign="left">
```

Comment:
I don't know what is the recommended convention for MUI when it comes to styling but I think it's easier to read if we put all css properties in the `sx`-prop. If there's only like one css property then it might be fine to inline. What do you think?

Not anything we need to change in this PR but I wanted to open up the discussion :)

Reply:
I don't care too much. I prefer the props variant when I can because it is easier to write.. :)